### ISSUE TITLE
Delete a specific photo from a collection

### ISSUE THREAD

**Issue by bumi** [bumi/tasveer#26]
In order to have exact control over the uploaded photos as a user I want to be able to delete a photo from a collection manually.

We have to make sure that photos do not get added again by the matching filter - so we have to keep it somehow in our local DB and mark it as deleted/ignored in a specific collection. 
The photo still could show up in another collection.

**Comment by bumi** [comment]
On this topic we also need to think about what happens to photos that get deleted on the device. 
Right now photos that get deleted on the device can no longer be shown within the app but are still on the server. 

I'd suggest we make sure that we show the photos within the app from a local cache/the server so it is independent of what happens to the photos in the iOS gallery.

**Comment by ludoded** [comment]
Good catch. I’m not caching photos right now, so I can show the photo from the server.
